Components/FlightControl/MEVManager.cpp

```diff
@@ -1,28 +1,19 @@
 #include "MEVManager.hpp"
 #include "PBBRxProtocolTask.hpp"
 #include "CommandMessage.hpp"
+#include "GPIO.hpp"
 
 MEVManager::MEVState MEVManager::shouldMevBeOpen = INDETERMINATE;
 
 void MEVManager::OpenMEV() {
     shouldMevBeOpen = OPEN;
-    PBBRxProtocolTask::SendPbbCommand(Proto::PbbCommand::Command::PBB_OPEN_MEV);
+    GPIO::MEV_EN::On();
 }
 
 void MEVManager::CloseMEV() {
     shouldMevBeOpen = CLOSE;
-    PBBRxProtocolTask::SendPbbCommand(Proto::PbbCommand::Command::PBB_CLOSE_MEV);
+    GPIO::MEV_EN::Off();
 }
 
 void MEVManager::HandleMEVTelemetry(Proto::TelemetryMessage& msg) {
-    if (shouldMevBeOpen == INDETERMINATE || !msg.has_combustionControlStatus()) {
-        // Do nothing
-        return;
-    }
-
-    if (!msg.combustionControlStatus().mev_open() && shouldMevBeOpen == OPEN) {
-        OpenMEV();
-    } else if (msg.combustionControlStatus().mev_open() && shouldMevBeOpen == CLOSE) {
-        CloseMEV();
-    }
-}
\ No newline at end of file
+}
```

Comment:
add a comment to this function to explain why you are still keeping it in

Reply:
it's gone now